jekyll/_docs/code-coverage.md

```diff
@@ -52,35 +52,39 @@ The [simplecov README](https://github.com/colszowka/simplecov/#getting-started)
 
 #### Scala
 
-In Scala 2.x you can use the [sbt-coverage](https://github.com/scoverage/sbt-scoverage) plugin.
+In Scala 2 you can use the [sbt-scoverage](https://github.com/scoverage/sbt-scoverage) plugin.
 
 Add the plugin to `project/plugins.sbt`.
 
 ```scala
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.0")
 ```
 
-Enable coverage in `circle.yml` with `sbt coverage test`. If you're building artifacts you may want to split running tests with coverage and producing the output assembly.
+For a simple project, enable coverage in `circle.yml` with `sbt coverage test:compile`.
 
 ```
 dependencies:
   override:
-    - echo "Skipping sbt test:compile, will compile during test coverage."
-test:
-  override:
-    - sbt coverage test
-    - sbt 'set test in assembly := {}' assembly
+    - sbt coverage test:compile
 ```
 
-Produce the coverage report and gather coverage artifacts.
+Run tests with coverage with `sbt coverage test` and produce a coverage report with `sbt coverageReport`.
```

Comment:
Remove "with coverage"; we're already operating in the context of that tool.

"Run tests with `sbt coverage test` and produce a coverage report with `sbt coverageReport`."